src/modules/units/components/UnitManagementTable.tsx

```diff
@@ -51,12 +49,12 @@ const UnitManagementTable: React.FC<Props> = ({
     return [
       UNIT_COLUMNS.unitType,
       UNIT_COLUMNS.unitId,
-      ...(unitGroupsEnabled ? [UNIT_COLUMNS.unitGroup] : []),
+      UNIT_COLUMNS.unitGroup,
```

Comment:
Suggest hiding this column when you're looking at 1 group already? That table are a bit squished and it is duplicative with unit type. I'm not sure about getting rid of unit type altogether, since they could technically be named differently. But it feels like we shouldn't need to show the Unit Group column when you're looking at the Unit Group page.

`...(unitGroupId ? [] : [UNIT_COLUMNS.unitGroup]),`